src/test/software/amazon/qldb/integrationtests/StatementExecutionIntegTest.java

```diff
@@ -185,23 +185,23 @@ public void execute_CreateIndex_IndexIsCreated() {
         String searchQuery = String.format(
             "SELECT VALUE indexes[0] FROM information_schema.user_tables WHERE status = 'ACTIVE' AND name ='%s'",
             Constants.TABLE_NAME);
-        String searchValue = driver.execute(
+        Result searchResult = driver.execute(
             txn -> {
-                Result result = txn.execute(searchQuery);
-
                 // Extract the index name by querying the information_schema.
                 /* This gives:
                 {
                     expr: "[MyColumn]"
                 }
                 */
-                String value = "";
-                for (IonValue row : result) {
-                    IonValue ionValue = ((IonStruct) row).get("expr");
-                    value = ((IonString) ionValue).stringValue();
-                }
-                return value;
+                return txn.execute(searchQuery);
             });
+
+
+        String searchValue = null;
```

Comment:
Could you check the other test cases and they also need it?

I see `execute_DeleteSingleDocument_DocumentIsDeleted` needs this

Reply:
This is scope creep and we have a ticket in the backlog for this.